test/mitmproxy/addons/test_proxyauth.py

```diff
@@ -109,6 +121,24 @@ def test_check():
         )
         assert not up.check(f)
 
+        ctx.configure(
```

Comment:
I think we should just [monkeypatch](https://docs.pytest.org/en/latest/monkeypatch.html) `ldap3.Connection` module so that we don't have any real connections but 100% code coverage. Does that work for you?